bin/tessel-2.js

```diff
@@ -353,6 +353,12 @@ makeCommand('update')
     flag: true,
     help: 'Update to the latest version regardless of current version.'
   })
+  .option('delete', {
+    abbr: 'd',
+    required: false,
+    flag: true,
+    help: 'Delete configuration files during update.'
+  })
```

Comment:
Yepp, agreed. Here the docs.. maybe just re-use it?
https://wiki.openwrt.org/doc/techref/sysupgrade

` -n do not save configuration over reflash`